jobs/integration/conftest.py

```diff
@@ -94,24 +98,40 @@ class Tools:
     """Utility class for accessing juju related tools"""
 
     def __init__(self, request):
+        self._request = request
         self.requests = aioify(obj=requests)
+
+    async def _load(self):
```

Comment:
I had to split this out to get the current Juju user, which was required for the fully-qualified model & offer names.